frontend/src/layouts/ProtectedLayout.vue

```diff
@@ -6,14 +6,19 @@ import SideNav, {
     type ISideNavItem,
 } from '@/components/common/SideNav.vue';
 import sideNavData from '@/static/sideNav.json';
-import { FAM_APPLICATION_ID } from '@/store/Constants';
+import { FAM_APPLICATION_ID, APP_ADMIN_ROLE } from '@/store/Constants';
 import { isApplicationSelected, selectedApplicationId } from '@/store/ApplicationState';
-
+import LoginUserState from '@/store/FamLoginUserState';
 
 const navigationData = ref<[ISideNavData]>(sideNavData as any);
 
 // Show and hide the correct sideNav btn based on the application
-const setSideNavAddPermissionOption = () => {
+const setSideNavOptions = () => {
+    // if user is app admin
+    if(LoginUserState.hasAccess(APP_ADMIN_ROLE)) {
```

Comment:
yeah, so this logic here has to change as well, which should be user is admin of the selected app

Reply:
I'll refactor this as well